jupyter/utils/install_pdf_deps.sh

```diff
@@ -1,20 +1,34 @@
-#!/bin/bash 
+#!/bin/bash
 
 # Install dependencies required for Notebooks PDF exports
 
-set -euxo
+set -euxo pipefail
+
+# Mapping of `uname -m` values to equivalent GOARCH values
+declare -A UNAME_TO_GOARCH
+UNAME_TO_GOARCH["x86_64"]="amd64"
+UNAME_TO_GOARCH["aarch64"]="arm64"
+UNAME_TO_GOARCH["ppc64le"]="ppc64le"
+UNAME_TO_GOARCH["s390x"]="s390x"
+
+ARCH="${UNAME_TO_GOARCH[$(uname -m)]}"
+if [[ -z "${ARCH:-}" ]]; then
+    echo "Unsupported architecture: $(uname -m)" >&2
+    exit 1
+fi
 
 # tex live installation
-echo "Installing TexLive to allow PDf export from Notebooks" 
-curl -L https://mirror.ctan.org/systems/texlive/tlnet/install-tl-unx.tar.gz -o install-tl-unx.tar.gz 
+echo "Installing TexLive to allow PDf export from Notebooks"
+curl -fL https://mirror.ctan.org/systems/texlive/tlnet/install-tl-unx.tar.gz -o install-tl-unx.tar.gz
 zcat < install-tl-unx.tar.gz | tar xf -
 cd install-tl-2*
 perl ./install-tl --no-interaction --scheme=scheme-small --texdir=/usr/local/texlive
-cd /usr/local/texlive/bin/x86_64-linux
+mv /usr/local/texlive/bin/"$(uname -m)-linux" /usr/local/texlive/bin/linux
+cd /usr/local/texlive/bin/linux
```

Comment:
@coderabbitai i'm not a fan of the move here; please offer alternative solutions of the problem for consideration

Reply:
@coderabbitai `/usr/local/texlive/bin/linux` is silly, can we install texlive to /usr/local/texlive/bin?